ClientApp/KachnaOnline/src/app/board-games/reservation-creation/reservation-creation.component.ts

```diff
@@ -10,6 +10,11 @@ import { Router } from "@angular/router";
 import { HttpStatusCode } from "@angular/common/http";
 import { BoardGamePageState, BoardGamesStoreService } from "../../shared/services/board-games-store.service";
 import { UserDetail } from "../../models/users/user.model";
+import { AuthenticationService } from "../../shared/services/authentication.service";
+import { Reservation } from "../../models/board-games/reservation.model";
+import { EMPTY, forkJoin, Observable, of } from "rxjs";
```

Comment:
If I am not mistaken, the import of `EMPTY` is not necessary, as it is not used anywhere.
```suggestion
import { forkJoin, Observable, of } from "rxjs";
```